spec/features/number_fields_spec.rb

```diff
@@ -1,14 +1,49 @@
 require "rails_helper"
 
-VALID_INTEGER_VALUES = ["2", "2,000"].freeze
+VALID_INTEGER_VALUES = ["2", "2,000", "1234   "].freeze
 INVALID_INTEGER_VALUES = ["two", "2 00", "2.37"].freeze
```

Comment:
Let's add some invalid integer and money values based on what we found in testing today? So things like "-20", "+20", "10_000" and "123.45e1" are the things me and Mas found.